pkg/platform/local/platform.go

```diff
@@ -108,7 +108,7 @@ func NewPlatform(ctx context.Context,
 
 	switch runtime.GOARCH {
 	case "arm64":
-		newPlatform.storeImageName = "gcr.io/iguazio/arm64v8/alpine:3.20"
+		newPlatform.storeImageName = "gcr.io/iguazio/alpine:3.20"
```

Comment:
why?

Reply:
@TomerShor reverted, I usually do this to run tests, because otherwise they don't work